lib/slack.py

```diff
@@ -32,30 +32,30 @@ def __init__(self):
         # Cache the client in memory
         self._bot_client = None
         self._user_client = None
-        self.clients = {'bot': self._bot_client, 'user': self._user_client}
+        self._clients = {'bot': self._bot_client, 'user': self._user_client}
 
     def send_message(self, params, type="Message"):
         # Sends message to the user/channel
         # Type should either be Message or Ephemeral
         if 'as_user' in params.keys() and params['as_user']:
-            client = self.get_client(type='user')
+            _client = self.get_client(type='user')
```

Comment:
This one is actually unnecessary since it's a temp variable within the function itself, not a class level variable